src/Widgets/NavigationWidget/SubComponents/MainNavigation.tsx

```diff
@@ -3,14 +3,19 @@ import Nav from 'react-bootstrap/Nav'
 import { NavItem as NavItem } from './NavItem'
 import { SearchBox } from './SearchBox'
 import { HomepageInstance } from '../../../Objs/Homepage/HomepageObjClass'
+import { NavigationWidgetInstance } from '../NavigationWidgetClass'
 
 export const MainNavigation = connect(function MainNavigation({
   root,
   searchInputLabel,
+  widget,
 }: {
   root: HomepageInstance
   searchInputLabel: string
+  widget: NavigationWidgetInstance
 }) {
+  if (widget.get('slimDesign')) return null
```

Comment:
* [ ] Adding a widget prop in order to access a condition the container could check makes not much sense. Move the condition up?